model/models.py

```diff
@@ -61,8 +61,7 @@ def forward(self, sub, rel):
 
 		sub_emb, rel_emb, all_ent	= self.forward_base(sub, rel, self.drop, self.drop)
 		obj_emb				= sub_emb + rel_emb
-
-		x	= self.p.gamma - torch.norm(obj_emb.unsqueeze(1) - all_ent, p=1, dim=2)		
+		x	= self.p.gamma - torch.norm(obj_emb.unsqueeze(1) - all_ent, p=1, dim=2)
```

Comment:
Please do not reformat code, as this will make it hard to review it. Could you remove all the formatting differences?